src/init.c

```diff
@@ -1290,7 +1306,8 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	if (mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC, NULL) == -1) {
+	if (system_mount("devpts", "/dev/pts", "devpts", MS_NOSUID| MS_NOEXEC,
+			 NULL) == -1) {
 		perror("mount devpts failed");
 		return -1;
 	}
```

Comment:
the commit for `Regrouped the early setup code (system mounts, becoming session leader and setting up a controlling terminal in a single function that is skipped when not running as "init".` is missing